frontend/assets/stylesheets/components/_asides.scss

```diff
@@ -44,3 +44,9 @@
     padding: rem($gs-baseline * 2);
     margin-bottom: rem($gs-baseline * 2);
 }
+
+.aside-feature {
+    @include mq(tablet) {
+        margin-top: rem(62px);
```

Comment:
As discussed I think we can get rid of this by introducing a `page-section__header`